react-app/components/Gallery/styles.css

```diff
@@ -7,6 +7,33 @@
 .gallery-content {
   display: flex;
   align-items: center;
+  justify-content: center;
+}
+
```

Comment:
It's possible to add ( in the future ) a code [formatter](https://prettier.io/).

I've seen many css files with different types of formatting, which is not consistent :thinking:

Reply:
Actually I'm using prettier, it will add [eslint](https://eslint.org/) for some rules